### PR TITLE
use locale weeks to match kinesis streams

### PR DESCRIPTION
**Pull request by daryl-mcmillan**
https://docs.aws.amazon.com/firehose/latest/dev/basic-deliver.html#es-index-rotation

### CODE REVIEW

File: test/tests.js [hunk 1]
```diff
@@ -63,10 +63,10 @@ describe( 'elasticsearch-rolling-index-name-parser', function() {
 			assert.equal( result.period, 'weekly', 'period should be weekly' );
 	
 			assert( moment.isMoment( result.startMoment ), 'start moment should be a moment' );
```
Comment (daryl-mcmillan) [inline test/tests.js]: US locale weeks start on Sunday vs ISO Monday, so most of these dates shift a day earlier.

File: test/tests.js [hunk 2]
```diff
@@ -93,10 +93,25 @@ describe( 'elasticsearch-rolling-index-name-parser', function() {
...
-			assert.equal( result.startMoment.toISOString(), '2008-12-29T00:00:00.000Z', 'should parse correct start moment' );
+			assert.equal( result.startMoment.toISOString(), '2008-12-28T00:00:00.000Z', 'should parse correct start moment' );
 			
 			assert( moment.isMoment( result.endMoment ), 'end moment should be a moment' );
-			assert.equal( result.endMoment.toISOString(), '2009-01-04T23:59:59.999Z', 'should construct correct end moment' );
+			assert.equal( result.endMoment.toISOString(), '2009-01-03T23:59:59.999Z', 'should construct correct end moment' );
+		} );
+
+		it( 'should start from Jan 1', function() {
```
Comment (daryl-mcmillan) [inline test/tests.js:167]: testing ISO vs US locale start week - ISO week 1 is the week of Jan 4th

File: test/tests.js [hunk 3]
```diff
@@ -99,6 +100,85 @@ describe( 'elasticsearch-rolling-index-name-parser', function() {
 			assert.equal( result.endMoment.toISOString(), '2009-01-04T23:59:59.999Z', 'should construct correct end moment' );
 		} );
 
+		it( 'should start from Jan 1', function() {
```
Comment (omsmith) [inline test/tests.js]:
```suggestion
		it( 'should start from Jan 4', function() {
```

File: package.json [hunk 4]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "elasticsearch-rolling-index-name-parser",
-  "version": "1.1.1",
+  "version": "2.0.0",
```
Comment (omsmith) [inline package.json]: Could just be 1.2 now with it being an option.

**Review by omsmith — DISMISSED** [review]

**Review by omsmith — APPROVED** [review]

**Pull request merged**